.github/workflows/deploy-mfe-s3.yml

```diff
@@ -63,7 +69,7 @@ jobs:
       run: cat dist/index.html
 
     - name: Share artifact inside workflow
-      uses: actions/upload-artifact@v3
+      uses: actions/upload-artifact@v4
```

Comment:
Omg v3 has been deprecated since November of 2024. I think we need to upgrade all the mfe actions with this change

![image](https://github.com/user-attachments/assets/a95a21e3-5c00-48c0-8598-4a6d0d6a11c9)